components/ScrollingAvatars/ScrollingAvatars.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC, useEffect, useState } from 'react';
+import Avatar from '@components/ScrollingAvatars/Avatar';
+import type { AvatarInfo } from './ScrollingAvatarsContainer';
+
+interface IProps {
+  /** pre-set width and height of each avatar */
+  size?: 'small' | 'mid' | 'large';
+  /** list of objects containing the configs for the avatars, maximum size is 24 */
+  avatarInfos: AvatarInfo[];
+  scrollDirection?: 'left' | 'right';
+  onMouseEnter?: () => void;
+  onMouseLeave?: () => void;
+  setName: (name: string) => void;
+  setPosition: (position: string) => void;
+}
+
+/** enum used to map size label and it's corresponding css properties.
+ * This enum must match that in ScrollingAvatar.scss */
+const AVATAR_ENUM = {
+  small: {
+    size: 60,
+    margin: 15,
+  },
+  mid: {
+    size: 100,
+    margin: 30,
+  },
+  large: {
+    size: 150,
+    margin: 50,
+  },
+};
```

Comment:
I don't like calling this an `enum` since this is a POJO enums are a real TS feature